includes/widgets/common.php

```diff
@@ -43,6 +43,51 @@ public function show_in_panel() {
 		return false;
 	}
 
+	/**
+	 * @param $shape String Shape name.
+	 *
+	 * @return string The shape path in the assets folder.
+	 */
+	private function get_shape_url( $shape ) {
+		return ELEMENTOR_ASSETS_URL . 'mask-shapes/' . $shape . '.svg';
+	}
+
+	/**
+	 * @return array Array of shapes with their URL as key.
+	 */
+	private function get_shapes() {
+		$shapes = [
+			$this->get_shape_url( 'circle' ) => __( 'Circle', 'elementor' ),
```

Comment:
Can't use the URL as a key because the site URL may be changed. use the name instead.
```suggestion
			'circle' => __( 'Circle', 'elementor' ),
```

Reply:
Why does it matter if the URL has changed? The key will get changed as well.